src/app/public/testing/public_api.ts

```diff
@@ -1,4 +1,8 @@
+export * from './dropdown/dropdown-testing.module';
 export * from './dropdown/dropdown-fixture';
 export * from './dropdown/popovers-fixture-dropdown';
 export * from './dropdown/popovers-fixture-dropdown-item';
 export * from './dropdown/popovers-fixture-dropdown-menu';
+
+export * from './popover/popover-testing.module';
+export * from './popover/popover-fixture';
```

Comment:
I'm noticing that the `app-extras.module.ts` file exports the component's `public_api` file, but doesn't export the `/testing/public_api` file. Is this right or should it be added there so consumers will have access to the test fixtures and their modules?

Reply:
Paul answered this one for me. It's an Angular convention.